db/seeds.rb

```diff
@@ -1,7 +1,41 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Author.destroy_all
+Author.create!([{
+  full_name: "Susan Cooper",
+  born: "in Burnham, Buckinghamshire, The United Kingdom ",
+  picture: "https://d.gr-assets.com/authors/1379606336p5/7308.jpg",
+  bio: "Susan Cooper was born in 1935, and grew up in England's Buckinghamshire, an area that was green countryside then but has since become part of Greater London. As a child, she loved to read, as did her younger brother,who also became a writer. After attending Oxford, where she became the first woman to ever edit that university's newspaper, Cooper worked as a reporter and feature writer for London's Sunday Times; her first boss was James Bond creator Ian Fleming.",
+  gender: "female"
+}, {
+  full_name: "Stephen King",
+  born: "in Portland, Maine, The United States",
+  picture: "https://d.gr-assets.com/authors/1362814142p5/3389.jpg",
+  bio: "Stephen Edwin King was born the second son of Donald and Nellie Ruth Pillsbury King. After his father left them when Stephen was two, he and his older brother, David, were raised by his mother.",
+  gender: "male"
+}])
+
+Genre.destroy_all
+Genre.create!([
+  { name: "Horror" },
+  { name: "Fantasy" }
+])
+
+Book.destroy_all
+Book.create!([{
+  title: "Greenwitch",
+  summary: "Simon, Jane, and Barney, enlisted by their mysterious great-uncle, arrive in a small coastal town to recover a priceless golden grail stolen by the forces of evil -- Dark.",
+  cover_image: "https://d.gr-assets.com/books/1353277730l/11588.c://d.gr-assets.com/books/1368431585l/24963.jpg",
+  author: Author.first,
+  genre_id: 2
+}, {
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.